docs/network-flow-visibility.md

```diff
@@ -56,14 +59,29 @@ ClickHouse as the data storage, and use Grafana as the data visualization and mo
 
 ### Deployment Steps
 
-To deploy the Grafana Flow Collector, the first step is to install the ClickHouse
-Operator, which creates, configures and manages ClickHouse clusters. Check the [homepage](https://github.com/Altinity/clickhouse-operator)
-for more information about the ClickHouse Operator. Current checked-in yaml is based on their
-[v0.18.2](https://github.com/Altinity/clickhouse-operator/blob/refs/tags/0.18.2/deploy/operator/clickhouse-operator-install-bundle.yaml) released version. Running the following command
-will install ClickHouse Operator into `kube-system` Namespace.
+We support deploying the Grafana Flow Collector with Helm. Here is the
+[Helm chart](../build/charts/theia/) for the Grafana Flow Collector. Please follow
+the instructions from the Helm chart [README](../build/charts/theia/README.md)
+to customize the installation.
+
+You can clone the repository and run the following command to install the Grafana
+Flow Collector into Namespace `flow-visibility`.
+
+```bash
+helm install -f <path-to-your-values-file> theia ./build/charts/theia -n flow-visibility --create-namespace
+```
+
+We recommend using Helm to deploy the Grafana Flow Collector. But if you prefer
```

Comment:
since e2e tests not longer mutates configMap as part of the run, do you think we should also switch to helm in jenkins and kind tests? also, maybe add a separate e2e job for local PV?

Reply:
In jenkins and kind tests, I think we can still use the YAML file and manifest generating scripts as Antrea side does now. As the manifest generating scripts are updated to use the `helm template` and it is easier to override values.yaml with the script. Adding a separate e2e job for local PV sounds feasible for me.